packages/atomic/src/components/atomic-smart-snippet/atomic-smart-snippet-suggestions.tsx

```diff
@@ -0,0 +1,258 @@
+ * @part answer-and-source - The wrapper around the answer and source of a related question.
+ * @part answer - The container displaying the full document excerpt of a related question's answer.
+ * @part footer - The wrapper around the source of a related question's answer.
+ * @part source-url - The URL to the document a related question's answer is extracted from.
+ * @part source-title - The title of the document a related question's answer is extracted from.
+ *
+ * @internal
+ */
+@Component({
+  tag: 'atomic-smart-snippet-suggestions',
+  styleUrl: 'atomic-smart-snippet-suggestions.pcss',
+  shadow: true,
+})
+export class AtomicSmartSnippetSuggestions implements InitializableComponent {
+  @InitializeBindings() public bindings!: Bindings;
+  public smartSnippetQuestionsList!: SmartSnippetQuestionsList;
+  @BindStateToController('smartSnippetQuestionsList')
+  @State()
+  public smartSnippetQuestionsListState!: SmartSnippetQuestionsListState;
+  public error!: Error;
+  @Element() public host!: HTMLElement;
+
+  /**
+   * The [heading level](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements) to use for the "People also ask" heading over the snippets, from 1 to 5.
+   *
+   * We recommend setting this property in order to improve accessibility.
+   *
+   */
+  @Prop({reflect: true}) public headingLevel = 0;
+
+  /**
+   * Sets the style of the snippets.
+   *
+   * Example:
+   * ```ts
+   * smartSnippet.snippetStyle = `
+   *   b {
+   *     color: blue;
+   *   }
+   * `;
+   * ```
+   */
+  @Prop({reflect: true}) snippetStyle?: string;
+
+  private id = randomID('atomic-smart-snippet-suggestions-');
+
+  public initialize() {
+    this.smartSnippetQuestionsList = buildSmartSnippetQuestionsList(
+      this.bindings.engine
+    );
+  }
+
+  private getRelatedQuestionId(relatedQuestion: SmartSnippetRelatedQuestion) {
+    // TODO: Change to the snippet's unique ID
+    return `${this.id}-${relatedQuestion.question
+      .toLowerCase()
+      .replace(/\s/g, '-')
+      .replace(/[^a-z-]/g, '')}`;
+  }
+
+  private get style() {
+    const styleTag = this.host
+      .querySelector('template')
+      ?.content.querySelector('style');
+    if (!styleTag) {
+      return this.snippetStyle;
+    }
+    return styleTag.innerHTML;
+  }
+
+  private renderQuestion(relatedQuestion: SmartSnippetRelatedQuestion) {
+    const toggleRelatedQuestion = () =>
+      relatedQuestion.expanded
+        ? this.smartSnippetQuestionsList.collapse(relatedQuestion.documentId)
+        : this.smartSnippetQuestionsList.expand(relatedQuestion.documentId);
+
+    return (
+      <Button
+        style="text-neutral"
+        part={
+          relatedQuestion.expanded
+            ? 'question-expanded-button'
+            : 'question-collapsed-button'
+        }
+        onClick={toggleRelatedQuestion}
+        class="flex items-center px-4"
+        ariaExpanded={`${relatedQuestion.expanded}`}
```

Comment:
out of curiosity, does this not work with the boolean without the back ticks?

Reply:
It doesn't, because Button is a functional component which takes a string as a parameter
![image](https://user-images.githubusercontent.com/54454747/166450694-42916ef0-0c74-4006-a4e1-ec5dae78b694.png)

as for the `button` element rendered by it, I'm not sure, I recall it not working so I didn't really try.

Reply:
yeah all aria-X attribute have to be strings